com.aobuchow.themes.spectrum/css/preference_styles.css

```diff
@@ -12,418 +12,400 @@
-		'breakpointIndicationColor=51,119,193'
-		'currentIPColor=90,90,90'
-		'infoIndicationColor=86,194,170'
-		'lineNumberColor=119,145,154'
-		'linked.slave.color=66,156,255'
-		'matchingTagIndicationColor=72,72,72'
-		'overrideIndicatorColor=78,120,117'
-		'searchResultHighlighting=false'
-		'searchResultIndication=true'
-		'searchResultTextStyle=BOX'
-		'secondaryIPColor=90,90,90'
-		'spellingIndicationColor=253,170,211'
-		'AbstractTextEditor.Color.Background=23,23,27'
-		'AbstractTextEditor.Color.FindScope=224,226,228'
-		'AbstractTextEditor.Color.Foreground=192,192,192'
-		'AbstractTextEditor.Color.SelectionBackground=117,117,117'
-		'AbstractTextEditor.Color.SelectionForeground=192,192,192'
-		'currentLineColor=47,57,60'
-		'deletionIndicationColor=224,226,228'
-		'filteredSearchResultIndicationColor=97,97,97'
-		'javaScriptOccurrenceIndicationColor=97,97,97'
-		'lineNumberColor=129,150,154'
-		'occurrenceIndicationColor=97,97,97'
-		'PHPReadOccurrenceIndicationColor=97,97,97'
-		'PHPWriteOccurrenceIndicationColor=97,97,97'
-		'printMarginColor=129,150,154'
-		'searchResultIndicationColor=97,97,97'
-		'writeOccurrenceIndicationColor=97,97,97'
+IEclipsePreferences#org-eclipse-ui-editors:org-eclipse-ui-themes-modern-dark-editors {
+  /* pseudo attribute added to allow contributions without replacing this node, see Bug 466075 */
+  preferences: 'AbstractTextEditor.Color.SelectionForeground.SystemDefault=false'
+    'AbstractTextEditor.Color.SelectionBackground.SystemDefault=false'
+    'AbstractTextEditor.Color.Background.SystemDefault=false'
+    'AbstractTextEditor.Color.Foreground.SystemDefault=false'
+    'hyperlinkColor.SystemDefault=false'
+    'hyperlinkColor=40,235,195'
+    'asOccurencesIndicationColor=72,72,72'
+    'breakpointIndicationColor=51,119,193'
+    'currentIPColor=90,90,90'
+    'infoIndicationColor=86,194,170'
+    'lineNumberColor=119,145,154'
+    'linked.slave.color=66,156,255'
+    'matchingTagIndicationColor=72,72,72'
+    'overrideIndicatorColor=78,120,117'
+    'searchResultHighlighting=false'
+    'searchResultIndication=true'
+    'searchResultTextStyle=BOX'
+    'secondaryIPColor=90,90,90'
+    'spellingIndicationColor=253,170,211'
+    'AbstractTextEditor.Color.Background=23,23,27'
+    'AbstractTextEditor.Color.FindScope=224,226,228'
+    'AbstractTextEditor.Color.Foreground=192,192,192'
+    'AbstractTextEditor.Color.SelectionBackground=117,117,117'
+    'AbstractTextEditor.Color.SelectionForeground=192,192,192'
+    'currentLineColor=47,57,60'
+    'deletionIndicationColor=224,226,228'
+    'filteredSearchResultIndicationColor=97,97,97'
+    'javaScriptOccurrenceIndicationColor=97,97,97'
+    'lineNumberColor=129,150,154'
+    'occurrenceIndicationColor=97,97,97'
+    'PHPReadOccurrenceIndicationColor=97,97,97'
+    'PHPWriteOccurrenceIndicationColor=97,97,97'
+    'printMarginColor=129,150,154'
+    'searchResultIndicationColor=97,97,97'
+    'writeOccurrenceIndicationColor=97,97,97';
```

Comment:
Note that a `;` was added. @pyvesb do you happen to know if this causes any issues? Everything seemed to be fine when I ran the plugin in child Eclipse after formatting the CSS, so I think it's okay?

Reply:
To be honest, I don't know for sure.

Reply:
@PyvesB its okay, this is why I’m going to merge this to the experimental branch first ;)